packages/react/src/Image.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "@hpcc-js/preact-shim";
+
+interface IImage {
+    x?: number;
+    y?: number;
+    height?: number;
+    width?: number;
+    href?: string;
+    text?: string;
+    yOffset?: number;
+}
+
+export const Image: React.FunctionComponent<IImage> = ({
+    x,
```

Comment:
Should x have a default value (should it be optional?)

Reply:
Yes I think so, that prevents ever seeing the svg "x is NaN" error.

Reply:
FYI I "think" React will spot an undefined and omit the entire assignment - so that wouldn't be the case.